lib/src/main/java/com/thenewboston/utils/PaginationOptions.kt

```diff
@@ -0,0 +1,30 @@
+package com.thenewboston.utils
+
+data class PaginationOptions(
+    val offset: Int?,
+    val limit: Int?
+) {
+    fun toQuery(): String {
+        return when {
+            offset == null || limit == null -> "?offset=0&limit=20"
+            else -> "?offset=$offset&limit=$limit"
+        }
+    }
+}
+
+object PAGE {
+    var DEFAULT: PaginationOptions = PaginationOptions(
+        offset = 0,
+        limit = 20
+    )
+
+    var PAGE_2: PaginationOptions = PaginationOptions(
+        offset = 20,
+        limit = 20
+    )
+
+    var THIRTY_ITEMS: PaginationOptions = PaginationOptions(
+        offset = 0,
+        limit = 30
+    )
```

Comment:
Should we not use `val` instead? `var` is used if we want to make it mutable as in change the value.

Reply:
Also not sure if we need to set default values for the SDK users. Just allow them to do this work and modify the parameters.